pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/TableRebalancer.java

```diff
@@ -140,6 +142,7 @@ public class TableRebalancer {
   // TODO: Consider making the timeoutMs below table rebalancer configurable
   private static final int TABLE_SIZE_READER_TIMEOUT_MS = 30_000;
   private static final int STREAM_PARTITION_OFFSET_READ_TIMEOUT_MS = 10_000;
+  private static final AtomicInteger _rebalanceJobCounter = new AtomicInteger(0);
```

Comment:
could you add a test in one of the integration tests to validate that this increments across all table rebalance objects? (you can just validate the gauge value -> check that if you create two table rebalance objects and call rebalance method then this is incremented by 2)